modules/javafx.graphics/src/main/java/javafx/scene/AccessibleRole.java

```diff
@@ -483,6 +483,7 @@ public enum AccessibleRole {
      * <ul>
      * <li> {@link AccessibleAction#SET_TEXT} </li>
      * <li> {@link AccessibleAction#SET_TEXT_SELECTION} </li>
+     * <li> {@link AccessibleAction#SHOW_TEXT_RANGE} </li>
```

Comment:
Thanks for catching this!

Only TextArea has a meaningful implementation, all other descendants of TextInputControl have a no-op (see `scrollCharacterToVisible()`, TextInputControlSkin:736)

We need to remove TextField mention from `AccessibleAction.SHOW_TEXT_RANGE`.

Reply:
TextField can be scrolled horizontally, so seems like even TextField should respond to SHOW_TEXT_RANGE.
So, instead of removing TextField mention from AccessibleAction.SHOW_TEXT_RANGE, a bug can/should be filed for missing implementation.
What do you think ?